<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtProjClass</class>
 <widget class="QMainWindow" name="QtProjClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtProj</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_main">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_cs2cs">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_src">
        <property name="title">
         <string>SRC</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_srcval">
         <item row="0" column="0">
          <widget class="QLabel" name="lab_srcCS">
           <property name="text">
            <string>cs:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="le_srcCS">
           <property name="placeholderText">
            <string>+proj=longlat +datum=WGS84 +no_defs</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lab_srcLat">
           <property name="text">
            <string>Lat(X)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="le_srcLat">
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lab_srcLon">
           <property name="text">
            <string>Lon(Y)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="le_srcLon">
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lab_srcH">
           <property name="text">
            <string>Height(Z)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="le_srcH">
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_dst">
        <property name="title">
         <string>DEST</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_dstval">
         <item row="0" column="0">
          <widget class="QLabel" name="lab_dstCS">
           <property name="text">
            <string>cs:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="le_dstCS">
           <property name="placeholderText">
            <string>+proj=longlat +datum=WGS84 +no_defs</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lab_dstLat">
           <property name="text">
            <string>Lat(X)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="le_dstLat">
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lab_dstLon">
           <property name="text">
            <string>Lon(Y)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="le_dstLon">
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lab_dstH">
           <property name="text">
            <string>Height(Z)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="le_dstH">
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTransfer"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTransfer">
   <property name="icon">
    <iconset resource="qtproj.qrc">
     <normaloff>:/image/play.png</normaloff>:/image/play.png</iconset>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
   <property name="toolTip">
    <string>Transfer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtproj.qrc"/>
 </resources>
 <connections/>
</ui>
